eureka-client/src/main/java/com/netflix/discovery/EurekaClient.java

```diff
@@ -94,23 +94,31 @@
     public InstanceInfo.InstanceStatus getInstanceRemoteStatus();
 
     /**
+     * @deprecated see {@link com.netflix.discovery.endpoint.EndpointUtils} for replacement
+     *
      * Get the list of all eureka service urls for the eureka client to talk to.
      *
      * @param zone the zone in which the client resides
      * @return The list of all eureka service urls for the eureka client to talk to.
      */
+    @Deprecated
```

Comment:
The PeerEurekaNodes on the server side also uses this method, so it should really not be dependent on EurekaClient for this functionality. Moving it to EndpointUtils was just a convenient way of moving this out of the client side interface. Feel free to enhance this to as part of the transport work.